components/heap/include/esp_heap_trace.h

```diff
@@ -114,10 +138,10 @@ esp_err_t heap_trace_stop(void);
  * - ESP_ERR_INVALID_STATE Heap tracing was already started.
  * - ESP_OK Heap tracing resumed.
  */
-esp_err_t heap_trace_resume(void);
+esp_err_t heap_trace_resume(void)
```

Comment:
this looks like an unintentional removal of the `;`. I'm not certain it would compile without it.

Reply:
thanks.

 I may of removed that in my last force-push

Reply:
updated.